src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import glob
+import h5py
+import argparse
+import sys
+import os
+import numpy as np
+import logging
+import matplotlib as mpl
+# The module name below depends on python version used to run the script
+# Imports Tkinter for python2, tkinter for python3
+try:
+    import Tkinter as tkinter
```

Comment:
I don't know how to install tkinter in the container with pip. Can you either figure that out or not use tkinter (I don't actually know what it is or does, so maybe you have to use it)?

Reply:
I think it can just be removed, I never needed tkinter to render animations.
It's used on line 261 below, but that whole else clause can probably go and perhaps be replaced by an assert somewhere.

Reply:
I can remove it.